packages/react/src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -525,26 +527,29 @@ class OverflowMenu extends Component {
     };
 
     return (
-      <ClickListener onClickOutside={this.handleClickOutside}>
-        <div
-          {...other}
-          role="menu"
-          aria-haspopup
-          aria-expanded={this.state.open}
-          className={overflowMenuClasses}
-          onKeyDown={this.handleKeyPress}
-          onBlur={this.handleBlur}
-          onClick={this.handleClick}
-          aria-label={ariaLabel}
-          id={id}
-          tabIndex={tabIndex}
-          ref={mergeRefs(ref, this.bindMenuEl)}>
-          <IconElement {...iconProps}>
-            {iconDescription && <title>{iconDescription}</title>}
-          </IconElement>
-          {open && wrappedMenuBody}
-        </div>
-      </ClickListener>
+      <React.Fragment>
+        <ClickListener onClickOutside={this.handleClickOutside}>
+          <div
+            {...other}
+            role="button"
+            aria-haspopup
+            aria-expanded={this.state.open}
+            className={overflowMenuClasses}
+            onKeyDown={this.handleKeyPress}
+            onBlur={this.handleBlur}
+            onClick={this.handleClick}
+            aria-label={ariaLabel}
+            id={id}
+            tabIndex={tabIndex}
+            ref={mergeRefs(ref, this.bindMenuEl)}>
+            <IconElement {...iconProps}>
+              {iconDescription && <title>{iconDescription}</title>}
+            </IconElement>
+            {open && wrappedMenuBody}
+          </div>
+        </ClickListener>
+        {menuBody}
```

Comment:
the menu body is being rendered twice, once in a `<FloatingMenu>` (`wrappedMenuBody` on L548) and once without (`menuBody`) on L551. but I don't think we can simply move the menu body and eliminate the floating menu wrapper without losing functionality like keyboard navigation